conf/cassandra.yaml

```diff
@@ -1251,6 +1251,8 @@ server_encryption_options:
   # Set to a valid keystore if internode_encryption is dc, rack or all
   keystore: conf/.keystore
   keystore_password: cassandra
+  outbound_keystore: conf/.keystore
```

Comment:
@jyothsnakonisa We have to also update support for this configuration in [PEMBasedSslContextFactory](https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/security/PEMBasedSslContextFactory.java), correct?

Reply:
@maulin-vasavada  thanks for pointing this out. I have updated the PR to make this change in `PEMBasedSslContextFactory`

Reply:
I don't have rights to resolve the discussion. Please do so for this.